Documentation/gettingstarted/gsg_starwars.rst

```diff
@@ -53,21 +53,23 @@ Each pod will be represented in Cilium as an :ref:`endpoint`. We can invoke the
     NAME           READY     STATUS    RESTARTS   AGE
     cilium-1c2cz   1/1       Running   0          26m
 
-    $ kubectl -n kube-system exec cilium-1c2cz cilium endpoint list
+    $ kubectl -n kube-system exec cilium-1c2cz -- cilium endpoint list
     ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                   IPv6                 IPv4            STATUS
                ENFORCEMENT        ENFORCEMENT
-    7624       Disabled            Disabled          9919       k8s:class=deathstar                           f00d::a0f:0:0:1dc8   10.15.185.9     ready
+    7624       Disabled            Disabled          9919      k8s:class=deathstar                           f00d::a0f:0:0:1dc8   10.15.185.9     ready
                                                                k8s:io.kubernetes.pod.namespace=default
                                                                k8s:org=empire
     10900      Disabled           Disabled          32353      k8s:class=xwing                               f00d::a0f:0:0:2a94   10.15.92.254    ready
                                                                k8s:io.kubernetes.pod.namespace=default
                                                                k8s:org=alliance
-    11010      Disabled            Disabled          9919       k8s:class=deathstar                           f00d::a0f:0:0:2b02   10.15.197.34    ready
+    11010      Disabled            Disabled          9919      k8s:class=deathstar                           f00d::a0f:0:0:2b02   10.15.197.34    ready
                                                                k8s:io.kubernetes.pod.namespace=default
                                                                k8s:org=empire
     50240      Disabled           Disabled          12904      k8s:class=tiefighter                          f00d::a0f:0:0:c440   10.15.28.62     ready
                                                                k8s:io.kubernetes.pod.namespace=default
                                                                k8s:org=empire
+    52921      Disabled           Disabled          4          reserved:health                               f00d::a0f:0:0:ceb9   10.15.126.89    ready
```

Comment:
Weird. We left align single digits but right align the others?

Reply:
Looks good when we render.

Reply:
shrug. I meant it's surprising that the CLI prints it out this way